azerothcore/azerothcore-wotlk
# fix(DB/Skinning): Firemane mobs not consistently dropping leather

## Changes Proposed:
-  Set groupids to 1 so firemane mobs always drop leather

## Issues Addressed:

- Closes #5821
- Closes chromiecraft/chromiecraft#632

## SOURCE:


## Tests Performed:

- See below


## How to Test the Changes:


1. Run ```SELECT * FROM `acore_world`.`skinning_loot_template` WHERE `entry` IN (4328, 4329, 4331, 4333, 4334);```
All the results should have a GroupId of 1
2. Learn skinning ```player learn $YOURNAME 50305.
Max skinning skill ```setskill 393 450```
Get a skinning knife ```additem 7005```
3. Use ```go creature 31215``` to kill and skin flamecallers and scalebanes. Every corpse should yield some sort of skinning loot.
4. Use ```go creature 31485``` to kill and skin scouts and ash tails. Every corpse should yield some sort of skinning loot.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
